Mathlib/Combinatorics/SimpleGraph/Path.lean

```diff
@@ -1087,6 +1087,11 @@ theorem mem_supp_iff (C : G.ConnectedComponent) (v : V) :
     v ∈ C.supp ↔ G.connectedComponentMk v = C :=
   Iff.rfl
 
+-- Correction of `SimpleGraph.ConnectedComponent.mem_supp_iff`
+@[simp]
+theorem mem_supp_iff' {G : SimpleGraph V} (C : G.ConnectedComponent) (v : V) :
```

Comment:
This version is due to the fact that while we have a `SetLike` instance for ConnectedComponent, we tend to explicitly use `c.supp` when talking about the vertices underlying a component. If you use `.supp`, you won't need this lemma.